constants/wram_constants.asm

```diff
@@ -91,6 +91,7 @@ DEF FONT_MASK EQU %00000111
 	const PSS_OPT          ; 7
 
 DEF ABILITIES_OPTMASK EQU 1 << ABILITIES_OPT
+DEF LINK_OPTMASK EQU (1 << NATURES_OPT) | (1 << ABILITIES_OPT) | (1 << PERFECT_IVS_OPT) | (1 << PSS_OPT)
 
 ; wInitialOptions2::
 DEF RESET_INIT_OPTS EQU 7
```

Comment:
```suggestion
; wInitialOptions2::
	const_def
	const EVS1_OPT        ; 0
	const EVS2_OPT        ; 1
	const_skip 5
	const RESET_INIT_OPTS ; 7
```
Can I define something like the above to reserve the EV bytes you mentioned? Since i've already accounted for them in the Link Options exchange.

Reply:
This would be great, yes.

Reply:
Accomplished in: a63f569